kernel/src/test/scala/org/apache/toree/kernel/protocol/v5/relay/ExecuteRequestRelaySpec.scala

```diff
@@ -88,7 +89,7 @@ class ExecuteRequestRelaySpec extends TestKit(
         // Expected does not actually match real return of magic, which
         // is a tuple of ExecuteReply and ExecuteResult
         val expected = new ExecuteAborted()
-        interpreterActorProbe.expectMsgClass(
+        interpreterActorProbe.expectMsgClass(max = Duration(5, TimeUnit.SECONDS),
```

Comment:
The default is 3 seconds, but it was flaky for me. This was another work-around that probably shouldn't be here.